pkg/bindings/network/network.go

```diff
@@ -12,12 +13,21 @@ import (
 )
 
 // Create makes a new CNI network configuration
-func Create(ctx context.Context, network *types.Network) (types.Network, error) {
+func Create(ctx context.Context, network *types.Network, createOptions *NetworkCreateOptions) (types.Network, error) {
```

Comment:
looks like you changed the wrong place, pkg/bindings is stable so this is a breaking change
pkg/domain is not stable so you can break it there and do not need to keep the old function

Reply:
I guess I didn't notice where the comment was and assumed I knew. Fixed